src/System.Console/src/FxCopBaseline.cs

```diff
@@ -0,0 +1,49 @@
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Microsoft.Reliability", "CA2002:DoNotLockOnObjectsWithWeakIdentity", Scope = "member", Target = "System.IO.SyncTextReader.#Dispose(System.Boolean)")]
```

Comment:
Did you file an issue to review these and actually apply them to the members with justification?